lbry/lbry/dht/node.py

```diff
@@ -28,6 +29,8 @@ def __init__(self, loop: asyncio.AbstractEventLoop, peer_manager: 'PeerManager',
         self.joined = asyncio.Event(loop=self.loop)
         self._join_task: asyncio.Task = None
         self._refresh_task: asyncio.Task = None
+        self._storage = storage
+        self._known_node_urls = known_node_urls or []
```

Comment:
(I know `known_node_urls` don't have to be passed on initialization, will rewrite my tests first to match the old signature of `start` first)